packages/frontend/src/redux/reducers/account/index.js

```diff
@@ -199,7 +199,16 @@ const account = handleActions({
                 loading: true
             }
         }
-    })
+    }),
+    [staking.getLockup]: (state, { error, payload, ready }) => 
+        (!ready || error)
+            ? {
+                ...state 
```

Comment:
Shouldn't we just return `state` here rather than shallow copying it?

Reply:
Right. Done.